projects/golang/go/Makefile

```diff
@@ -64,7 +64,7 @@ clean:
 
 .PHONY: install-deps
 install-deps:
-	yum update -y && yum install -y which git rpmdevtools awscli rpm-build golang-1.15.14-1.amzn2.0.1 pcre-devel glibc-static hostname procps-ng
+	yum update -y && yum install -y which git rpmdevtools go-srpm-macros awscli rpm-build golang-1.15.14-1.amzn2.0.1 pcre-devel glibc-static hostname procps-ng
```

Comment:
i think for the 1.16 build we should install the 1.16 yum package? or from upstream if it doesn't exist?

Reply:
will address this in a following PR